sites/site-utilities/src/definitions/native/html-native.definition.ts

```diff
@@ -0,0 +1,80 @@
+import { DataType } from "@microsoft/fast-tooling";
+import { WebComponentAttribute } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { voidElements } from "@microsoft/fast-tooling/dist/data-utilities/html-element";
+
+/**
+ * These definitions are dependent on a devDependency on "vscode-web-custom-data"
+ * speficially the browsers.html-data.json which defines native html elements and their attributes
+ * as used by vscode
+ */
+import * as vcodeHTMLData from "vscode-web-custom-data/data/browsers.html-data.json";
+
+const valueSets: any[] = vcodeHTMLData.valueSets;
+
+export const htmlNativeDefinitions: any = {
+    version: 1, // WebComponentDefinition has this typed as 1, vcodeHTMLData.version could be anything
+    tags: vcodeHTMLData.tags,
+};
+
+function getDataTypeForAttribute(attribute: any): DataType {
+    /**
+     * For some reason valueSets references "v" in many boolean like attributes but does not define "v",
+     * It does define "b" and both are utilized as booleans for all attributes
+     */
+    switch (attribute.valueSet) {
+        case "b":
+        case "v":
```

Comment:
If you define this as an interface (as I suggested above), you can then set up something like:

```ts
export interface VSCodeAttribute {
    valueSet: "b" | "v"; // add whichever strings are available for valueSet
}
```

Reply:
Do we want to define the 25 or so strings that are in the definition file for this when we only care about 2?

Reply:
I think leaving it at 2 with a note on where these can be found in the git repository and that this is only a subset would be fine.

Reply:
done and done